jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -153,7 +153,7 @@ def user_api(self):
             self,
             expose=(
                 'align_by', 'wcs_fast_approximation',
-                'delete_subsets', 'viewer', 'orientation',
+                'viewer', 'orientation', 'delete_subsets',

```

Comment:
Why the reorder here?

Reply:
i removed it initially then added it back in out of order, ill change it back